README.md

```diff
@@ -38,7 +38,7 @@ Default options:
 :html       => false,  # Use ANSI color codes rather than HTML.
 :multiline  => true,   # Display in multiple lines.
 :plain      => false,  # Use colors.
-:raw        => false,  # Do not recursively format object instance variables.
+:raw        => true,   # Recursively format object instance variables.
```

Comment:
I dont think we need to change defaults, since this is a breaking change and brokes the semantic versioning.